AppStream.patch

```diff
@@ -0,0 +1,55 @@
+From be8b02a0ac92d6df03840607a67baea7d8e8bef8 Mon Sep 17 00:00:00 2001
+From: AsciiWolf <mail@asciiwolf.com>
+Date: Mon, 19 Oct 2020 22:34:54 +0200
+Subject: [PATCH] Add AppStream metadata
+
+---
+ gui/CMakeLists.txt     |  1 +
+ gui/chiaki.appdata.xml | 26 ++++++++++++++++++++++++++
+ 2 files changed, 27 insertions(+)
+ create mode 100644 gui/chiaki.appdata.xml
+
+diff --git a/gui/CMakeLists.txt b/gui/CMakeLists.txt
+index 7072b83..7eda103 100644
+--- a/gui/CMakeLists.txt
++++ b/gui/CMakeLists.txt
+@@ -103,3 +103,4 @@ install(TARGETS chiaki
+ 		BUNDLE DESTINATION bin)
+ install(FILES chiaki.desktop DESTINATION share/applications)
+ install(FILES chiaki.png DESTINATION share/icons/hicolor/512x512/apps)
++install(FILES chiaki.appdata.xml DESTINATION share/metainfo)
+diff --git a/gui/chiaki.appdata.xml b/gui/chiaki.appdata.xml
+new file mode 100644
+index 0000000..75a34df
+--- /dev/null
++++ b/gui/chiaki.appdata.xml
+@@ -0,0 +1,26 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<!-- Copyright 2020 Florian Märkl <info@florianmaerkl.de> -->
++<component type="desktop">
++  <id>chiaki.desktop</id>
++  <name>Chiaki</name>
++  <summary>Free and Open Source Client for PlayStation 4 Remote Play</summary>
++  <metadata_license>CC0-1.0</metadata_license>
++  <project_license>GPL-3.0</project_license>
++  <developer_name>Florian Märkl</developer_name>
++  <url type="homepage">https://github.com/thestr4ng3r/chiaki</url>
++  <description>
++    <p>
++      Chiaki is a Free and Open Source Client for PlayStation 4 Remote Play. It can be used to play in real time on a PlayStation 4 as long as there is a network connection.
```

Comment:
Are usage instructions also displayed in the app? https://github.com/thestr4ng3r/chiaki#usage

Reply:
Good point! I will update the appdata file. :-)

Reply:
Done. ;-)